documentation/source/rotating_logger.rst

```diff
@@ -0,0 +1,26 @@
+.. _rotating-logger:
+
+***********************
+Rotating Disk Log Files
+***********************
```

Comment:
```suggestion
*****************
Log File Rotation
*****************
```
I'm not sure "disk" is a useful word here.

Reply:
Done.